Logger/BaseLoggerMixins.cs

```diff
@@ -2,6 +2,25 @@
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger bLogger, string message,params string[] args)
+        {
+            if (bLogger == null) { throw new System.ArgumentNullException("The BaseLogger Parameter is null"); }
```

Comment:
The string parameter to `ArgumentNullException` is expected to the [name of the argument](https://docs.microsoft.com/en-us/dotnet/api/system.argumentnullexception.-ctor?view=netcore-3.0#System_ArgumentNullException__ctor_System_String_) that was null.
```suggestion
            if (bLogger == null) { throw new System.ArgumentNullException(nameof(bLogger)); }
```